src/helper/FormattedDate.ts

```diff
@@ -0,0 +1,24 @@
+export class myFormattedDate {
+  static convertFormattedDate = () => {
+    const date = new Date();
+    const day = date.getDate();
+    const monthIndex = date.getMonth();
+    const year = date.getFullYear();
+    const minutes = date.getMinutes();
+    const hours = date.getHours();
+    const seconds = date.getSeconds();
+    const myFormattedDate =
```

Comment:
Please use string formatter or use dayjs
```suggestion
    const myFormattedDate = `${year}-${monthIndex + 1}-${day} ${hours}:${minutes}:${seconds}`
    ```
or use dayjs
```suggestion
dayjs(new Date()).format('DD/MM/YYYY HH:mm:ss')
```